app/scripts/components/modeldetails.js

```diff
@@ -112,6 +113,9 @@ var exports = (function () {
         e.stopPropagation();
         $(e.target).closest(".panel").children(".collapse").collapse("toggle");
       },
+      getActiveModelData: function (str) {
```

Comment:
It looks like you want to be able to use `_.get` in the template. 
Use lodash from your template like this: http://stackoverflow.com/questions/37694243/using-lodash-in-all-of-vue-component-template